discovery-provider/plugins/pedalboard/scripts/sandbox.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Sandbox file you can use to just test run an app!
+ */
+
+import App from "../src/app"
+import { Table, Tracks, Users } from "../src/models"
+import { server } from "./server"
+
+export type SharedData = {
+    userUpdateCount: number,
+    trackUpdateCount: number,
+    counterCount: number,
+}
+
+const main = async () => {
+    await new App<SharedData>({
+        userUpdateCount: 0,
+        trackUpdateCount: 0,
+        counterCount: 0,
+    })
+        .listen<Tracks>("tracks", async (app, msg) => {
+            console.log(`got track update! ${JSON.stringify(msg)}`)
+            app.updateAppData((data) => {
+                return {
+                    ...data,
+                    trackUpdateCount: data.trackUpdateCount += 1,
+                }
+            })
+        })
+        .listen<Users>("users", async (app, msg) => {
+            console.log(`got user update! ${JSON.stringify(msg)}`)
+            app.updateAppData((data) => {
+                return {
+                    ...data,
+                    userUpdateCount: data.userUpdateCount += 1,
+                }
+            })
+        })
+        .repeat(10000, async (app) => {

```

Comment:
unclear to developer what 10000 means here. maybe nice to export some tip level timing constants? or maybe vendor dayjs in so you can say  https://day.js.org/docs/en/durations/durations 
```
.repeat({seconds: 20})
.repeat({years: 5})
```